### ISSUE TITLE
GitHub app to streamline PR review process

### ISSUE THREAD

**Issue by SPGoding** [SpyglassMC/Spyglass#1198]
Would be nice to have a GitHub app that does:

* Add "pr:blocked" label to Pull Requests that indicate dependencies on other PRs (through keywords like "based on #42", "base on #42", "blocked by #42") until all dependencies have been merged.
* Add "pr:mature" label to Pull Requests that have been open for at least 48 hours and the last commit was made at least 24 hours ago.

This should make it easier at a glance to see which PRs can be merged and which ones can be reviewed. More labels could be added to further suit our process.

I can provide hosting for the app.
